### DISCUSSION TITLE
How to get the Galaxy Brain badge?

### DISCUSSION THREAD

**Discussion by alisenola** [community/community#77134] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello, everybody.

I hope you are going well.

As matter of fact, I'd like to ask a question, just how to get the **Galaxy Brain** badge?

Would you please help me?

If there is someone who has already experienced that, please don't hesitate to reach out to me here.

Crossing my fingers to hear from  you at an early date.

Warm regards.

**Comment by NeekuroVonz** [comment]
To obtain **Galaxy Brain** badge, you have to answer 2 questions in any discussion topic, and it have to be marked as **Answered** by the Owner of that discussion topic.

**Comment by ghost** [comment]
video link: https://youtu.be/oJWFfp3sGYA?si=QrEtH_hy36r5xoZ8

This vid helped me to understand how to gain badges in github. Hope it helps you

**Comment by pecraveiro** [comment]
@worldbestpro @NeekuroVonz @Yuheshpandian to achieve the 'Galaxy Brain' achievement, take note of the following:

- Answer at least 2 discussions (got an accepted answer).

1. Take note, you CANNOT mark your own answers to your own questions. They have to be marked by another user.
2. Discussions should be in a public repository.
3. There's a current bug observed by some users whereby marking your own answers can prevent you from getting the badge instead.

Hope this answer has helped you, and all the best in getting the achievement! :)

Reference: https://github.com/drknzz/GitHub-Achievements

By the way, could you help me get it by marking my answer!

**Reply by ghost** [reply]
Yeah youre right

**Comment by upmorpheus** [comment, marked as answer]
To earn the Galaxy Brain badge on GitHub, you need to fulfil the following criteria:

- Answer at least 2 discussions in any repository, excluding your own repositories.
- Get your answers marked as the best answer by the person who asked the question.

Note that you cannot answer your own questions or mark your own answers as the best answer. Your answers should be relevant to the discussion topic and provide helpful insights or solutions.

To increase your chances of earning the Galaxy Brain badge, actively participate in discussions that align with your expertise and provide thoughtful and informative responses. Be respectful of other users and offer constructive feedback when appropriate.

Here are some additional tips for getting the Galaxy Brain badge:

- Identify active and engaging discussions by checking the "Trending" tab on the discussions page.
- Carefully read the discussion topic and ensure your answer addresses the specific question or issue raised.
- Provide well-structured and easy-to-understand responses, using proper grammar and formatting.
- Support your answers with relevant links, code snippets, or references if applicable.
- Be patient and persistent. It may take some time to accumulate the required two accepted answers.

By following these guidelines and actively engaging in discussions, you can increase your chances of earning the Galaxy Brain badge and demonstrating your expertise within the GitHub community.

**Comment by alisenola** [comment]
Wow, I get the whole picture, mate.

Thanks.

**Comment by AlpianPPLG** [comment]
### Hi There👋 

### 🧠 How to Earn the Galaxy Brain Badge on GitHub

To unlock the **Galaxy Brain** achievement badge on your GitHub profile, you need to actively assist the community by answering questions in GitHub Discussions. 

Here are the specific, official requirements to earn and level up this badge:

#### 📋 The Core Requirements
* **Answer Public Discussions:** You must provide a helpful response to a question within the **Discussions** tab of any public repository.
* **Get Marked as the Best Answer:** The author of the discussion (or a repository maintainer) must click the **"Mark as answer"** button on your specific response.
* **External Repositories Only:** The discussions must be in repositories owned by other users or organizations. Answering questions in your own personal repositories does not count toward the achievement.

---

#### 📈 Badge Levels (Tiers)
The Galaxy Brain badge evolves and changes its appearance as you continue to help more developers across the platform:

| Tier | Required Accepted Answers |
| :--- | :--- |
| **Bronze (Level 1)** | 2 accepted answers |
| **Silver (Level 2)** | 8 accepted answers |
| **Gold (Level 3)** | 16 accepted answers |
| **Platinium (Level 4)** | 32 accepted answers |

---

#### 💡 Quick Tips to Get Your Answers Selected
1. **Target Q&A Categories:** Look for repositories that have the **Discussions** feature explicitly enabled with a dedicated "Q&A" or "Questions" category.
2. **Be Clear and Scannable:** Use clean Markdown formatting, code blocks, and bullet points so your solution is incredibly easy for the original poster to read and test.
3. **Provide Context:** Don't just paste code blocks; briefly explain *why* your solution works. Thorough, insightful answers are much more likely to be officially marked as the accepted answer!

**Comment by longyijdos** [comment]
Confirmed in September 2026: answers in your **own public repository do count** toward the Galaxy Brain achievement.

I tested this in my own repository:

* Another collaborator created the Discussion questions.
* I replied to them.
* The collaborator marked two of my replies as accepted answers.
* I received the Galaxy Brain achievement immediately after the second accepted answer.

So the repository itself does **not** need to be owned by someone else. The important part seems to be that the questions are asked by another user and your answers are accepted by another user.
